htmlbasic/bootstrap2.html/homework2-1.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
+</head>
+<body>
+
+<div class="jumbotron text-center">
+  <h1>Alexa's Cafe Menu Friday</h1>
+  <p>We have a selection of American dishes to enjoy!</p>
+
+<div id="myCarousel" class="carousel slide" data-ride="carousel">
+  <!-- Indicators -->
+  <ol class="carousel-indicators">
+    <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+    <li data-target="#myCarousel" data-slide-to="1"></li>
+    <li data-target="#myCarousel" data-slide-to="2"></li>
+  </ol>
+
+  <!-- Wrapper for slides -->
+  <div class="carousel-inner">
+    <div class="item active">
+      <img src="https://media-cdn.tripadvisor.com/media/photo-s/07/a9/fa/d6/cafe-arcadia.jpg" alt="Our Home">
+    </div>
+
+    <div class="item">
+      <img src="https://i1.cdn.stylehub.io/56868/b72189dc8f8f492e250e363a03f51ebf/x/o/sectorlanding-cafes-contact2.jpg" alt="Our Pride">
+    </div>
+
+    <div class="item">
+      <img src="https://media-cdn.tripadvisor.com/media/photo-s/12/b6/9b/12/the-idea-of-84-cafe-is.jpg" alt="Our Life">
+    </div>
+  </div>
+
+  <!-- Left and right controls -->
+  <a class="left carousel-control" href="#myCarousel" data-slide="prev">
```

Comment:
none of the controllers is working. This is probably because of the many html errors reported by the validator